.github/actions/setup/action.yaml

```
@@ -29,6 +29,11 @@ runs:
       shell: bash
       run: curl -o /usr/local/bin/uds -L https://github.com/defenseunicorns/uds-cli/releases/download/v${{ inputs.udsCliVersion }}/uds-cli_v${{ inputs.udsCliVersion }}_Linux_${{steps.setUdsCliArch.outputs.ARCH}} && chmod +x /usr/local/bin/uds
 
+    - name: Install Lula
+      uses: defenseunicorns/lula-action/setup@095636b7880051e11b05f10a582fdd911526161c
+      with:
+        version: v0.3.0
```

```suggestion
    - name: Install Lula
      uses: defenseunicorns/lula-action/setup@095636b7880051e11b05f10a582fdd911526161c
      with:
        # renovate: datasource=github-tags depName=defenseunicorns/lula versioning=semver-coerced
        version: v0.3.0
```

Also is there a tag on that action repo? Usually we pin to that in case there are breaking changes on an update we get a notice from the semantic version.  See this: https://github.com/defenseunicorns/uds-common/blob/d50b4ebe3712fea4114289db0c36da61a5f60eb5/.github/workflows/tag-release.yaml#L36

---

Not yet, its out of scope for our EoQ goal but is on the radar to do.